C#/C#.sublime-syntax

```diff
@@ -1541,7 +1546,7 @@ contexts:
 
   finally_block:
     - meta_scope: meta.block.trycatch.cs
-    - match: (finally)\b\s*
+    - match: finally\b
```

Comment:
Are you sure removing the `\b` anchor doesn't introduce a regression? I didn't check how the context flow goes.